docs/OPERATIONS.md

```diff
@@ -0,0 +1,101 @@
+# Deploy a Strategy / Vault for V2
+
+NOTE: This repo is encouraged to create multiple scripts for governance and dev multisig execution of complex transactions
+https://github.com/iearn-finance/chief-multisig-officer
+
+**IMPORTANT**: you should **NOT** create a new release with a test vault
+
+## Before deploying
+- [ ] Coordinate with Core Dev strategist for getting a review in board https://github.com/orgs/iearn-finance/projects/5
+- [ ] Peer review completed by:
+    - yearn.rocks: at least 2 strategists
+    - Experimental: at least 1 strategist + 1 core dev in board 
+- [ ] Check if want token has a deploy vault already (>=v0.3.0) and coordinate to use that first if possible.
+- [ ] (only if a new vault is needed) Deploy using the registry.
+    - `registry.newExperimentalVault(want, you, multisig, treasury, "", "")`
+        - args: token, gov, guardian, rewards, name, symbol
+        - multisig (strategists) = '0x16388463d60FFE0661Cf7F1f31a7D658aC790ff7'
+        - gov for testing = '0x846e211e8ba920b353fb717631c015cf04061cc9' (dev.ychad.eth) 
+        - treasury = '0x93A62dA5a14C80f265DAbC077fCEE437B1a0Efde'
+        (treasury.ychad.eth)
+    - [ ] Coordinate with core devs proper cap limit for new vault and other settings
+    - [ ] Check new vault has ABI setup on etherscan (If Verification with vyper and proxy is not working that is ideal)
+    - [ ] Set a deposit limit to the vault
+    vault.setDepositLimit(50_000 * 1e18)
+    $50k USD converted to your want. Example above is 50k DAI
+- [ ] Deploy strategy with settings and upload code to Etherscan for verification
+- [ ] Tag github review issue https://github.com/orgs/iearn-finance/projects/5 with deployed version. Add mainnet addresses
+
+## After deploying strategy
+- [ ] Add strategy to vault
+    - `vault.addStrategy(strategy, debt_ratio, rate_limit, 1000)`
+    - debt_ratio should be 9800 if first strategy on vault
+    - rate_limit is 0 unless there is reason for it to be different
+
+- [ ] Set keeper
+    - `strategy.setKeeper(keep3r_manager)`
+    - keep3r_manager = '0x13dAda6157Fee283723c0254F43FF1FdADe4EEd6'
+- [ ] Set rewards
+    - `strategy.setRewards(address)`
+    Read below if you want to use the sharer contract
+- [ ] Set management fee to 0
+    - `vault.setManagementFee(0)`
+- [ ] Set governance
+    - `vault.setGovernance(multisig)`
+    - multisig = '0x846e211e8ba920b353fb717631c015cf04061cc9'
+    (dev.ychad.eth)
+    - Governance needs to be accepted before it is in place. After you set this you will still have power until part 2.
+- [ ] Run tests against "live" vault and strategy in mainnet-fork
```

Comment:
Can we describe a set of standard tests here that we want to make sure work as they should for every vault/strategy?

Reply:
I see some tests are described further below, perhaps reference them in this section?

Reply:
Tests described and also op tests added to the brownie-mix :)